src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -1119,23 +1158,10 @@ export class BundleDataClient {
     });
     start = performance.now();
 
-    // Go through expired deposits in this bundle and now prune those that we have seen a fill for to construct
-    // the list of expired deposits we need to refund in this bundle.
-    expiredBundleDepositHashes.forEach((relayDataHash) => {
```

Comment:
We can no longer handle expired deposits here because we need to check that the fill status for these deposits is not Filled already